libpng.yaml

```diff
@@ -51,3 +51,6 @@ subpackages:
       runtime:
         - libpng
     description: libpng dev
+
+advisories:
```

Comment:
I'd like to solve this particular issue in `wolfictl` — I wouldn't have expected this to show up... 🤔